protokube/pkg/protokube/volume_mounter.go

```diff
@@ -52,6 +52,12 @@ func (k *VolumeMountController) mountMasterVolumes() ([]*Volume, error) {
 	}
 
 	for _, v := range attached {
+		// Do not ever try to mount root devices
+		if v.LocalDevice == "/dev/sda1" || v.LocalDevice == "/dev/xvda" {
```

Comment:
This is was put in just in case. The filtering should take care of the problem, but we should never mount a root volume ;)